### DISCUSSION TITLE
Make `Notification::send()` return `void`

### DISCUSSION THREAD

**Discussion by titantwentyone** [filamentphp/filament#5755] [General]
This is really a follow up to PR #5732 and something I found while digging into this.

The `send` method of `Notification` returns itself for chaining. Because of this, the following is possible and is a false positive:

```
//this passes but no notification is called from the method
livewire(SomeComponent::class)
    ->call('some_method_which_doesnt_fire_a_notification')
    ->assertNotified(
        Notification::make()
            ->title('I dont exist!')
            ->danger()
            ->send()
    );
```
I originally thought this was an error in the docs and that we had to send the notification to assert it but you're actually just creating one within the test so assertions will succeed.

As `send` should be the last in the chain (anything after this will not be stored in the session), could/should this possibly return `void` so that a type error will occur when the above happens? Realise this is possibly breaking, but something for v3?

**Comment by danharrin** [comment]
I don't think the change should be made, you might want to chain `sendToDatabase()` after for example.
